### ISSUE TITLE
#2 Space out the Nav Bar Options

### ISSUE THREAD

**Issue by dianetle** [dianetle/bluebean#2]
I don't want the menu options centered, but some more space between them would be nice.

**Comment by LisaHong** [comment]
Hello! May I work on this issue?

**Comment by dianetle** [comment]
> Hello! May I work on this issue?

Yes you can. I just assigned the issue to you. Thank you for helping!
